django_ledger/forms/transactions.py

```diff
@@ -56,12 +65,9 @@ def __init__(self, *args, entity_slug, ledger_pk, user_model, je_pk=None, **kwar
             user_model=self.USER_MODEL
         )
 
-        if je_pk:
-            je_qs = je_qs.filter(uuid__exact=self.JE_PK)
-
         for form in self.forms:
             form.fields['account'].queryset = account_qs
-            form.fields['journal_entry'].queryset = je_qs
```

Comment:
Can you elaborate the purpose of this change? If you remove the queryset then you leave the door open to possible assignment of JEs that are not related to this entity. I can see the initial, but if queryset is defined, the initial is derived from the actual je associated with the transaction.

Reply:
should be good to go now